### ISSUE TITLE
move to using webdev to develop

### ISSUE THREAD

**Issue by devoncarew** [dart-lang/dart-pad#1069]
We currently use `grind serve-custom-backend` to build and serve the app when developing; we should probably move to using `webdev serve`; it has a faster iteration cycle.

**Comment by domesticmouse** [comment]
I'm wondering how to integrate the custom build steps that grind runs into `webdev serve`. Any hints?

**Comment by devoncarew** [comment]
I haven't looked into it, but perhaps:
- is it possible to drive custom build steps from `build_runner`?
- do we still need the custom steps (I think it's largely used to concatenate files for the polymer UI)?

**Comment by domesticmouse** [comment]
I've done an initial investigation, and it looks like this is blocked on a bunch of other work.

I tried removing `vulcanize` as a first step towards being able to wrap up tasks in `build_runner` compliant format, but we need `vulcanize` to remove http imports from our source, which is being removed from Chrome in M75.

I tried doing a `grind build` followed by `webdev`, but that winds up throwing errors in webdev's injected `client.js`.

In short, getting this codebase clean enough to work with webdev is going to require a lot of yak shaving.

**Comment by RedBrogdon** [comment]
By "http imports," do you mean HTML imports? Are these the ones for Polymer we're talking about, in other words?

**Comment by domesticmouse** [comment]
Yup

**Comment by parlough** [comment]
Since with https://github.com/dart-lang/dart-pad/issues/2808 we're using Flutter to build the UI, we've been using its tooling appropriately and are no longer using `build_runner` or `webdev`.
